### PR TITLE
optimized stylesheet. Removed DRY code

### PR DESCRIPTION
**Pull request by ghost**
First ever commit.
Combined selector with similar properties. Moved Tag stylesheets to top.

### CODE REVIEW

File: resources/web/style.css [hunk 1]
```diff
@@ -8,28 +63,27 @@
 
 .container {
     position:relative;
-    padding: 0 0 0 25px;
+    padding:0 0 0 25px;
```
Comment (Hackebein) [inline resources/web/style.css]: In properties, put a space after the : character.

File: resources/web/style.css [hunk 2]
```diff
@@ -39,103 +93,581 @@
...
-#fontawesome {
-    width: 30px;
+.yellowPill {
+    background-color: rgb(255, 235, 0);
+    color: black;
 }
 
-#spacer {
-    margin-left: 15px;
+.purplePill {background-color: rgb(100, 66, 166);}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 3]
```diff
@@ -39,103 +93,581 @@
...
+
+.redPill {background-color: #990000;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 4]
```diff
@@ -39,103 +93,581 @@
...
+
+.greenPill {background-color: green;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 5]
```diff
@@ -39,103 +93,581 @@
...
+
+.bluePill {background-color: blue;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 6]
```diff
@@ -39,103 +93,581 @@
...
+    vertical-align: middle; 
+    padding: 2px 4px 2px 4px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 5px; 
 }
 
-#middle {
-    display: inline-block;
+.purplePill-sm {background-color: rgb(100, 66, 166);}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 7]
```diff
@@ -39,103 +93,581 @@
...
+
+.redPill-sm {background-color: #990000;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 8]
```diff
@@ -39,103 +93,581 @@
...
+
+.greenPill-sm {background-color: green;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 9]
```diff
@@ -39,103 +93,581 @@
...
+
+.bluePill-sm {background-color: blue;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 10]
```diff
@@ -39,103 +93,581 @@
...
     text-align: center;
+    color: #6441a5;
     padding-top: 10px;
     padding-bottom: 10px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.alert-danger {background-color: red;}
```
Comment (Hackebein) [inline resources/web/style.css:308]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 11]
```diff
@@ -39,103 +93,581 @@
...
+
+.alert-success {background-color: white;}
```
Comment (Hackebein) [inline resources/web/style.css:310]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 12]
```diff
@@ -39,103 +93,581 @@
...
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.alert-danger {background-color: red;}
+
+.alert-success {background-color: white;} 
+
+.alert-info {background-color: #add8e6;}
```
Comment (Hackebein) [inline resources/web/style.css:312]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 13]
```diff
@@ -1,3 +1,58 @@
...
+    color: #aaaaaa;
+    font-size: 14px;
+}
+tr:nth-child(odd) {
+    font-family: "Roboto", Sans-serif;
+    background: rgba(0, 0, 0, 0.3);
+    color: #aaaaaa;
+    font-size: 14px;
+}
+th, td {padding: 4px;}
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector, property and closing braces

File: resources/web/style.css [hunk 14]
```diff
@@ -39,103 +93,581 @@
...
+    color: #eee;
+    background: rgba(183, 80, 234, 0.5) !important;
+    border: none !important;
+    outline: none !important;
 }
 
-table {
-    border: 0px;
-    width: 100%;
+.yellowPill, .purplePill, .redPill, .greenPill, .bluePill,
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector

File: resources/web/style.css [hunk 15]
```diff
@@ -39,103 +93,581 @@
...
+.purplePill-sm, .redPill-sm, .greenPill-sm, .bluePill-sm {
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector

File: resources/web/style.css [hunk 16]
```diff
@@ -39,103 +93,581 @@
...
+    vertical-align: middle;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 5px;
 }
 
-option:hover {
-    background-color:#cd201f;
-    cursor: pointer;
+.yellowPill, .purplePill, .redPill, .greenPill, .bluePill {
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector

File: resources/web/style.css [hunk 17]
```diff
@@ -39,103 +93,581 @@
...
-    margin-left: 15px;
+.purplePill {background-color: rgb(100, 66, 166);}
+
+.redPill {background-color: #990000;}
+
+.greenPill {background-color: green;}
+
+.bluePill {background-color: blue;}
+
+.purplePill-sm, .redPill-sm, .greenPill-sm, .bluePill-sm {
```
Comment (Hackebein) [inline resources/web/style.css]: use one line per selector

File: resources/web/style.css [hunk 18]
```diff
@@ -39,103 +93,581 @@
...
+    outline: none !important;
+}
+
+/* Bootstrap and JQuery Section */
+
+.ui-slider .ui-slider-handle { 
+    visibility: hidden;
+}
+
+.alert-danger, .alert-success, .alert-info{
```
Comment (Hackebein) [inline resources/web/style.css:292]: use one line per selector

File: resources/web/style.css [hunk 19]
```diff
@@ -39,103 +93,581 @@
...
+    -moz-border-radius: 0px;
+    border-radius: 0px;
+    background: #6136b1;
+}
+.ui-tabs-vertical .ui-tabs-panel {
+    padding: 1em;
+    float: left;
+    width: 55em;
+}
+.ui-tabs-vertical .ui-tabs-nav .ui-state-hover, .ui-state-focus {
```
Comment (Hackebein) [inline resources/web/style.css:381]: use one line per selector

File: resources/web/style.css [hunk 20]
```diff
@@ -39,103 +93,581 @@
...
+    border-color: #6136b1;
+    background: rgb(56, 56, 56);
+}
+.ui-tabs-anchor:active, .ui-tabs-anchor:focus {
```
Comment (Hackebein) [inline resources/web/style.css:385]: use one line per selector

File: resources/web/style.css [hunk 21]
```diff
@@ -39,103 +93,581 @@
...
+
+.tooltip.bottom .tooltip-arrow {
+    border-bottom-color: #444488;
+}
+
+.tooltip.left .tooltip-arrow {
+    border-left-color: #444488;
+}
+
+input, textarea {
```
Comment (Hackebein) [inline resources/web/style.css:445]: use one line per selector

File: resources/web/style.css [hunk 22]
```diff
@@ -39,103 +93,581 @@
...
+
+.btn:hover {
+    font-color: #eeeeee;
+    color: #eeeeee;
+    background: rgba(150, 150, 150, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+}
+
+.btn:active, .btn.active {
```
Comment (Hackebein) [inline resources/web/style.css:471]: use one line per selector

File: resources/web/style.css [hunk 23]
```diff
@@ -39,103 +93,581 @@
...
+    font-color: #cccccc;
+    color: #cccccc;
+    background: rgba(100, 100, 100, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn.active.focus, .btn.active:focus, .btn.focus,
```
Comment (Hackebein) [inline resources/web/style.css:480]: use one line per selector

File: resources/web/style.css [hunk 24]
```diff
@@ -39,103 +93,581 @@
...
+.btn:active.focus, .btn:active:focus, .btn:focus {
```
Comment (Hackebein) [inline resources/web/style.css:481]: use one line per selector

File: resources/web/style.css [hunk 25]
```diff
@@ -39,103 +93,581 @@
...
+    outline: none !important;
+}
+
+.btn-primary:hover {
+    background: rgba(150, 150, 150, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+}
+
+.btn-primary:active, .btn-primary.active {
```
Comment (Hackebein) [inline resources/web/style.css:503]: use one line per selector

File: resources/web/style.css [hunk 26]
```diff
@@ -39,103 +93,581 @@
...
+    background: rgba(100, 100, 100, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn-primary.active.focus, .btn-primary.active:focus, .btn-primary.focus,
```
Comment (Hackebein) [inline resources/web/style.css:510]: use one line per selector

File: resources/web/style.css [hunk 27]
```diff
@@ -39,103 +93,581 @@
...
+.btn-primary:active.focus, .btn-primary:active:focus, .btn-primary:focus {
```
Comment (Hackebein) [inline resources/web/style.css:511]: use one line per selector

File: resources/web/style.css [hunk 28]
```diff
@@ -39,103 +93,581 @@
...
+}
+
+.btn-default:active, .btn-default.active {
+    background: rgba(100, 100, 100, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn-default.active.focus, .btn-default.active:focus, .btn-default.focus,
```
Comment (Hackebein) [inline resources/web/style.css:538]: use one line per selector

File: resources/web/style.css [hunk 29]
```diff
@@ -39,103 +93,581 @@
...
+.btn-default:active.focus, .btn-default:active:focus, .btn-default:focus {
```
Comment (Hackebein) [inline resources/web/style.css:539]: use one line per selector

File: resources/web/style.css [hunk 30]
```diff
@@ -39,103 +93,581 @@
...
+}
+
+.btn-warning:active, .btn-warning.active {
+    background: rgba(255, 69, 0, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn-warning.active.focus, .btn-warning.active:focus, .btn-warning.focus,
```
Comment (Hackebein) [inline resources/web/style.css:574]: use one line per selector

File: resources/web/style.css [hunk 31]
```diff
@@ -39,103 +93,581 @@
...
+.btn-warning:active.focus, .btn-warning:active:focus, .btn-warning:focus {
```
Comment (Hackebein) [inline resources/web/style.css:575]: use one line per selector

File: resources/web/style.css [hunk 32]
```diff
@@ -39,103 +93,581 @@
...
+}
+
+.btn-success:active, .btn-success.active {
+    background: rgba(0, 255, 0, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn-success.active.focus, .btn-success.active:focus, .btn-success.focus,
```
Comment (Hackebein) [inline resources/web/style.css:602]: use one line per selector

File: resources/web/style.css [hunk 33]
```diff
@@ -39,103 +93,581 @@
...
+.btn-success:active.focus, .btn-success:active:focus, .btn-success:focus {
```
Comment (Hackebein) [inline resources/web/style.css:603]: use one line per selector

File: resources/web/style.css [hunk 34]
```diff
@@ -39,103 +93,581 @@
...
+    outline: none !important;
+}
+
+.btn-danger:hover {
+    background: rgba(255, 20, 20, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+}
+
+.btn-danger:active, .btn-danger.active {
```
Comment (Hackebein) [inline resources/web/style.css:623]: use one line per selector

File: resources/web/style.css [hunk 35]
```diff
@@ -39,103 +93,581 @@
...
+    background: rgba(255, 0, 0, 0.5) !important;
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.btn-danger.active.focus, .btn-danger.active:focus, .btn-danger.focus,
```
Comment (Hackebein) [inline resources/web/style.css:630]: use one line per selector

File: resources/web/style.css [hunk 36]
```diff
@@ -39,103 +93,581 @@
...
+.btn-danger:active.focus, .btn-danger:active:focus, .btn-danger:focus {
```
Comment (Hackebein) [inline resources/web/style.css:631]: use one line per selector

**Review by Hackebein — CHANGES_REQUESTED** [review]
CSS code style changes

File: resources/web/style.css (hunk 13 above)
Comment (ghost) [inline resources/web/style.css]: The history marks are pretty bad on this one though. I can't see your changes really. It looks weird. Is therer a link to the full document containing your changes?

For example. I didn't even touch .container but it shows up on your changes. Im new to github, pls help 💃

Comment (Hackebein) [reply resources/web/style.css]: i make no changes, i comment only all lines they need a change.
All together: use one line per selector, property and closing braces
for example replace
```css
th, td {padding: 4px;}
```
with
```css
th,
td {
    padding: 4px;
}
```

Special on line 11:
In properties, put a space after the : character.
replace
```css
padding:0 0 0 25px;
```
with
```css
padding: 0 0 0 25px;
```

**Comment by BrandenB** [comment]
I hope this was tested. Going to merge for now.

**Pull request merged**